content/en/docs/gke/deploy/delete-cli.md

```diff
@@ -1,39 +1,88 @@
 +++
 title = "Delete using CLI"
-description = "Deleting Kubeflow from GCP using the command line interface (CLI)"
-weight = 6
+description = "Deleting Kubeflow from Google Cloud using the command line interface (CLI)"
+weight = 7
```

Comment:
For my education, what is this `weight` used for?

Reply:
It's for ordering docs in the same folder.